src/main/java/com/amazon/opendistroforelasticsearch/security/OpenDistroSecurityPlugin.java

```diff
@@ -896,22 +892,6 @@ public Settings additionalSettings() {
             settings.add(Setting.groupSetting(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_CONFIG_ENDPOINTS + ".",  Property.NodeScope));
             settings.add(Setting.intSetting(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_THREADPOOL_SIZE, 10, Property.NodeScope, Property.Filtered));
             settings.add(Setting.intSetting(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_THREADPOOL_MAX_QUEUE_LEN, 100*1000, Property.NodeScope, Property.Filtered));
-            settings.add(Setting.boolSetting(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_LOG_REQUEST_BODY, true, Property.NodeScope, Property.Filtered));
-            settings.add(Setting.boolSetting(ConfigConstants.OPENDISTRO_SECURITY_AUDIT_RESOLVE_INDICES, true, Property.NodeScope, Property.Filtered));
```

Comment:
Removing registration of audit logging params. These must be migrated to new audit.yml. 
Cluster startup will fail if they are provided. This is desired as part of deprecating the old way of specifying config in elasticsearch.yml.

Reply:
Other than being consistent with `roles.yml`, `internal_users.yml`, what is the benefit of deprecating the old way of specifying audit config settings?

Reply:
Audit configuration will always be loaded from index. User might be confused why the settings are different/ignored from elasticsearch.yml it is starting up with.

Reply:
Does the same apply to audit.yml? Users might be confused that the actual settings are different/ignored. IMO, it will be better to gradually deprecate those settings.

Reply:
I feel the users will be assume the same behavior as others roles.yml, internal_users.yml ...